src/opera/pge/rtc_s1/rtc_s1_pge.py

```diff
@@ -171,26 +171,61 @@ def _core_filename(self, inter_filename=None):
 
         return self._cached_core_filename
 
-    def _rtc_filename(self, inter_filename, use_validity_start_time=False):
+    def _core_static_filename(self, inter_filename=None):
+        """
+        Returns the core file name component for static layer products produced
+        by the RTC-S1 PGE.
+
+        The core file name component for RTC-S1 products consists of:
+
+        <Core filename>-STATIC
+
+        Where <Core filename> is returned by RtcS1PostProcessorMixin._core_filename().
+
+        Parameters
+        ----------
+        inter_filename : str, optional
+            The intermediate filename of the output product to generate the
+            core filename for. This parameter may be used to inspect the file
+            in order to derive any necessary components of the returned filename.
+            Once the core filename is cached upon first call to this function,
+            this parameter may be omitted.
+
+        Returns
+        -------
+        core_static_filename : str
+            The core file name component to assign to static layer products
+            created by this PGE.
+
+        """
+        core_filename = self._core_filename(inter_filename)
+
+        return f"{core_filename}-STATIC"
+
+    def _rtc_filename(self, inter_filename, static_layer_product=False):
         """
         Returns the base file name to use for RTC products produced by this PGE.
 
         The base filename for the RTC PGE consists of:
 
             <Core filename>_<BURST ID>_<ACQUISITION TIME>_<PRODUCTION TIME>_<SENSOR>_<SPACING>_<PRODUCT VERSION>
 
-        Where <Core filename> is returned by RtcS1PostProcessorMixin._core_filename().
+        Where <Core filename> is returned by RtcS1PostProcessorMixin._core_filename()
+        if static_layer_product is False, otherwise it the value returned by
+        RtcS1PostProcessorMixin._core_static_filename()
+
+        If static_layer_product is True, <ACQUISITION TIME> will correspond
+        to the configured data validity start time (as defined in the RunConfig).
 
         Parameters
         ----------
         inter_filename : str
```

Comment:
I know this is not part of the changes, but to me this description is not very clear.